embark install error - Ubuntu

### Bug Report

#### Summary

<!-- Provide a summary describing the problem you are experiencing. -->

#### Current behavior

embark is not installing in Ubuntu

<!--
  What is the current (buggy) behavior? Feel free to add as much information as possible.
  Also, a screenshot often says more than a thousand words. :)
  -->

#### How to reproduce

<!--
Provide steps to reproduce the bug.
Adding a failing Unit or Functional Test would help us a lot - you can submit one in a Pull Request separately, referencing this bug report.
-->

#### Expected behavior

embark should install properly
<!-- What was the expected (correct) behavior? -->

#### If you encounter an error, please create a logfile using the following command and post the output here

```
binwayne@binwayne:~$ sudo npm install -g embark
npm WARN deprecated istanbul@0.4.5: This module is no longer maintained, try this instead:
npm WARN deprecated   npm i nyc
npm WARN deprecated Visit https://istanbul.js.org/integrations for other alternatives.
npm WARN deprecated ipfs-api@17.2.4: ipfs-api has been renamed to ipfs-http-client, please update your package.json to get the latest version.
npm WARN deprecated fs-promise@2.0.3: Use mz or fs-extra^3.0 with Promise Support
npm WARN deprecated tar.gz@1.0.7: ⚠️  WARNING ⚠️ tar.gz module has been deprecated and your application is vulnerable. Please use tar module instead: https://npmjs.com/tar

> scrypt@6.0.3 preinstall /usr/local/lib/node_modules/embark/node_modules/scrypt
> node node-scrypt-preinstall.js

Error: Error: Command failed: ./configure
./configure: line 1904: config.log: Permission denied
./configure: line 1914: config.log: Permission denied


> scrypt@6.0.3 preinstall /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/scrypt
> node node-scrypt-preinstall.js

Error: Error: Command failed: ./configure
./configure: line 1904: config.log: Permission denied
./configure: line 1914: config.log: Permission denied


> typechecker@2.0.8 preinstall /usr/local/lib/node_modules/embark/node_modules/extract-opts/node_modules/typechecker
> node ./cyclic.js


> typechecker@2.0.8 preinstall /usr/local/lib/node_modules/embark/node_modules/extendr/node_modules/typechecker
> node ./cyclic.js

/usr/local/bin/embark -> /usr/local/lib/node_modules/embark/bin/embark

> keccak@1.4.0 install /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/keccak
> npm run rebuild || echo "Keccak bindings compilation fail. Pure JS implementation will be used."


> keccak@1.4.0 rebuild /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/keccak
> node-gyp rebuild

gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/keccak/build'
gyp ERR! System Linux 4.15.0-46-generic
gyp ERR! command "/usr/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/keccak
gyp ERR! node -v v10.15.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! keccak@1.4.0 rebuild: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the keccak@1.4.0 rebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Keccak bindings compilation fail. Pure JS implementation will be used.

> scrypt@6.0.3 install /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/scrypt
> node-gyp rebuild

gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/scrypt/build'
gyp ERR! System Linux 4.15.0-46-generic
gyp ERR! command "/usr/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/scrypt
gyp ERR! node -v v10.15.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 

> secp256k1@3.5.2 install /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/secp256k1
> npm run rebuild || echo "Secp256k1 bindings compilation fail. Pure JS implementation will be used."


> secp256k1@3.5.2 rebuild /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/secp256k1
> node-gyp rebuild

gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/secp256k1/build'
gyp ERR! System Linux 4.15.0-46-generic
gyp ERR! command "/usr/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/secp256k1
gyp ERR! node -v v10.15.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! secp256k1@3.5.2 rebuild: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the secp256k1@3.5.2 rebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Secp256k1 bindings compilation fail. Pure JS implementation will be used.

> sha3@1.2.2 install /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/sha3
> node-gyp rebuild

gyp ERR! configure error 
gyp ERR! stack Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/sha3/build'
gyp ERR! System Linux 4.15.0-46-generic
gyp ERR! command "/usr/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/sha3
gyp ERR! node -v v10.15.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
npm WARN remix-tests@0.1.1 requires a peer of yo-yoify@latest but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.4.0 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.
npm WARN acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/embark/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: scrypt@6.0.3 (node_modules/embark/node_modules/ganache-cli/node_modules/ganache-core/node_modules/scrypt):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: scrypt@6.0.3 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sha3@1.2.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the sha3@1.2.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/binwayne/.npm/_logs/2019-03-29T12_56_45_846Z-debug.log

```

#### Please provide additional information about your system

**OS**: Ubuntu 18.04
**Embark Version**: 
**Node Version**: 10.15.3
**NPM Version**:6.9.0

#### Sometimes issues are related to Embark's installation. Can you provide information on how Embark was installed?

--- a/packages/embark-utils/package.json
+++ b/packages/embark-utils/package.json
@@ -49,7 +49,7 @@
     "bip39": "2.5.0",
     "clipboardy": "1.2.3",
     "colors": "1.3.2",
-    "ethereumjs-wallet": "0.6.0",
+    "ethereumjs-wallet": "0.6.3",
     "embark-i18n": "^4.1.0-beta.0",
     "follow-redirects": "1.5.7",
     "fuzzy": "0.1.3",

--- a/packages/embark/package.json
+++ b/packages/embark/package.json
@@ -128,7 +128,7 @@
     "eth-ens-namehash": "2.0.8",
     "ethereumjs-tx": "1.3.7",
     "ethereumjs-util": "6.0.0",
-    "ethereumjs-wallet": "0.6.0",
+    "ethereumjs-wallet": "0.6.3",
     "file-loader": "2.0.0",
     "find-up": "2.1.0",
     "flatted": "0.2.3",

